core/src/main/java/jenkins/security/OverriddenAuthorizationStrategy.java

```diff
@@ -0,0 +1,172 @@
+    }
+
+    /**
+     * @return wrapped {@link AuthorizationStrategy}
+     */
+    @Nonnull
+    public AuthorizationStrategy getBaseStrategy() {
+        return baseStrategy;
+    }
+
+    @Nonnull
+    private <T> ACL overrideACL(@Nonnull final ACL baseAcl, @Nonnull final T item, @Nonnull final Class<T> klazz) {
+        return new ACL() {
+            @Override
+            public boolean hasPermission(@Nonnull Authentication a, @Nonnull Permission p) {
+                for (AuthorizationStrategyOverrideBase<? super T> d: AuthorizationStrategyOverrideConfiguration.enables(klazz)) {
+                    Boolean permission = d.hasPermission(item, a, p);
+                    if (permission != null) {
+                        return permission.booleanValue();
+                    }
+                }
+                return baseAcl.hasPermission(a, p);
+            }
+        };
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Collection<String> getGroups() {
+        return getBaseStrategy().getGroups();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ACL getRootACL() {
+        Jenkins j = Jenkins.getInstanceOrNull();
+        if (j == null) {
+            return getBaseStrategy().getRootACL();
+        }
+        return overrideACL(getBaseStrategy().getRootACL(), j, Jenkins.class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @SuppressWarnings("deprecation")
+    @Override
+    public @Nonnull ACL getACL(@Nonnull AbstractProject<?,?> project) {
+        return overrideACL(getBaseStrategy().getACL(project), project, AbstractProject.class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public @Nonnull ACL getACL(@Nonnull Job<?,?> project) {
+        return overrideACL(getBaseStrategy().getACL(project), project, Job.class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public @Nonnull ACL getACL(@Nonnull View item) {
+        return overrideACL(getBaseStrategy().getACL(item), item, View.class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public @Nonnull ACL getACL(@Nonnull AbstractItem item) {
+        return overrideACL(getBaseStrategy().getACL(item), item, AbstractItem.class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public @Nonnull ACL getACL(@Nonnull User user) {
+        return overrideACL(getBaseStrategy().getACL(user), user, User.class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public @Nonnull ACL getACL(@Nonnull Computer computer) {
+        return overrideACL(getBaseStrategy().getACL(computer), computer, Computer.class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public @Nonnull ACL getACL(@Nonnull Cloud cloud) {
+        return overrideACL(getBaseStrategy().getACL(cloud), cloud, Cloud.class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public @Nonnull ACL getACL(@Nonnull Node node) {
```

Comment:
This approach is pretty unreliable. If somebody creates new getACL()  methods in Auth strategy, he should ensure that these methods get overridden in this class.